src/FacadeAccountCreation/FacadeAccountCreation.Core/Models/Connections/ConnectionPersonModel.cs

```diff
@@ -1,5 +1,8 @@
-﻿namespace FacadeAccountCreation.Core.Models.Connections
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace FacadeAccountCreation.Core.Models.Connections
 {
```

Comment:
For consistency with the other files, no need to apply { } on namespace scope. It's not mandatory but just for more of consistency.

Reply:
Since we don't want to make too many changes to the non-test classes, I suggest we leave this for another time.